### ISSUE TITLE
noaa-ghe SNS topic request

### ISSUE THREAD

**Issue by vcabbage** [awslabs/open-data-registry#328]
Would it be possible to have an SNS topic setup for the noaa-ghe data (https://registry.opendata.aws/noaa-ghe/)?

Thank you!

**Comment by zflamig** [comment]
Yes. We are following up on the status of this. Thanks for the ping!

**Comment by zflamig** [comment]
@vcabbage We should be keeping all the GHE data indefinitely at this point in time
